packages/core/protocol-config.json

```diff
@@ -130,6 +141,20 @@
       "network_registry_contract_address": "0xf0D4D48866C4F9665212B29A64c14A1f0FEDFD4c",
       "tags": ["etherscan"],
       "network_registry_proxy_contract_address": "0x15e068Ef1f76319b1848b1fcB3e49D68724AEE07"
+    },
+    "monte_rosa": {
+      "network_id": "xdai",
+      "environment_type": "production",
+      "version_range": "1.89||1.90",
+      "channel_contract_deploy_block": 0,
+      "token_contract_address": "0x978D91ddFdb0c6eaCC22A258fE498957a79c5F4C",
+      "channels_contract_address": "0xCe7209FD81ED129C2ea8369715c168419e4148Ef",
```

Comment:
this is the same address as in Paleochora, in Valencia we need to make sure a new HoprChannels is created

Reply:
Once the new details are generated by the CI when the next release is created, we will put this information here.